packages/fxa-react/components/LoadingSpinner/index.stories.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import LoadingSpinner from './index';
+
+storiesOf('components/LoadingSpinner', module).add('basic', () => (
+  <LoadingSpinner />
```

Comment:
Getting this in storybook:

<img width="1674" alt="image" src="https://user-images.githubusercontent.com/13018240/83066887-5a76e280-a02b-11ea-829c-10d595377ad1.png">